src/components/forms/siteurl.js

```diff
@@ -34,9 +34,11 @@ export default class SiteURLForm extends Component {
               name="url"
               value={url}
               placeholder="URL of your Netlify site"
-              autocapitalize="off"
+              autocapi
```

Comment:
Whoa, this doesn't seem right.

Reply:
oh whattt happened here?

Reply:
I must have done that on accident.  Redoing the format completely didn't do that.